### PR TITLE
qt5: update to 5.15.3+20210429.

### PR DESCRIPTION
**Pull request by Johnnynator**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/qt5/template [hunk 1]
```diff
@@ -21,8 +20,8 @@ short_desc="Cross-platform application and UI framework (QT5)"
 maintainer="John <me@johnnynator.dev>"
 license="GPL-3.0-or-later, LGPL-3.0-or-later"
 homepage="https://qt.io/"
-distfiles="http://download.qt.io/official_releases/qt/${version%.*}/${version}/single/qt-everywhere-src-${version}.tar.xz"
-checksum=3a530d1b243b5dec00bc54937455471aaa3e56849d2593edb8ded07228202240
+distfiles="https://void.johnnynator.dev/distfiles/qt5-${version}.tar.gz"
```
Comment (ericonr) [inline srcpkgs/qt5/template:23]: Is it feasible to document how the tarball was generated?

Comment (Johnnynator) [reply srcpkgs/qt5/template:27]: Sure, done. The current one is created with slightly different ignored submodules, but I didn't want to regenerate it just to save like 5mb.

Comment (ericonr) [reply srcpkgs/qt5/template:27]: Nice! Thanks a lot.

(Maybe we can package the archive-all thingy? Haven't looked how complex it is yet).

Comment (Johnnynator) [reply srcpkgs/qt5/template:27]: Afaik it only needes Python, so it should be trivial.

**Pull request merged**